spanner/system-test/spanner.test.js

```diff
@@ -53,103 +53,139 @@ test.after.always(async (t) => {
+  t.regex(output, new RegExp(`Created database ${DATABASE_ID} on instance ${INSTANCE_ID}.`));
 });
 
 // insert_data
 test.serial(`should insert rows into an example table`, async (t) => {
-  let output = await tools.runAsync(`${crudCmd} insert ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
-  t.true(output.includes(`Inserted data.`));
+  const results = await tools.runAsyncWithIO(`${crudCmd} insert ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
+  const output = results.stdout + results.stderr;
+  t.regex(output, new RegExp(`Inserted data.`));
 });
 
 // query_data
 test.serial(`should query an example table and return matching rows`, async (t) => {
-  const output = await tools.runAsync(`${crudCmd} query ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
-  t.true(output.includes(`SingerId: 1, AlbumId: 1, AlbumTitle: Go, Go, Go`));
+  const results = await tools.runAsyncWithIO(`${crudCmd} query ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
+  const output = results.stdout + results.stderr;
+  t.regex(output, new RegExp(`SingerId: 1, AlbumId: 1, AlbumTitle: Go, Go, Go`));
 });
 
 // read_data
 test.serial(`should read an example table`, async (t) => {
-  const output = await tools.runAsync(`${crudCmd} read ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
-  t.true(output.includes(`SingerId: 1, AlbumId: 1, AlbumTitle: Go, Go, Go`));
+  const results = await tools.runAsyncWithIO(`${crudCmd} read ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
+  const output = results.stdout + results.stderr;
+  t.regex(output, new RegExp(`SingerId: 1, AlbumId: 1, AlbumTitle: Go, Go, Go`));
 });
 
 // add_column
 test.serial(`should add a column to a table`, async (t) => {
-  const output = await tools.runAsync(`${schemaCmd} addColumn ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
-  t.true(output.includes(`Waiting for operation to complete...`));
-  t.true(output.includes(`Added the MarketingBudget column.`));
+  const results = await tools.runAsyncWithIO(`${schemaCmd} addColumn ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
+  const output = results.stdout + results.stderr;
+  t.regex(output, new RegExp(`Waiting for operation to complete...`));
+  t.regex(output, new RegExp(`Added the MarketingBudget column.`));
 });
 
 // update_data
 test.serial(`should update existing rows in an example table`, async (t) => {
-  let output = await tools.runAsync(`${crudCmd} update ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
-  t.true(output.includes(`Updated data.`));
+  const results = await tools.runAsyncWithIO(`${crudCmd} update ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
+  const output = results.stdout + results.stderr;
+  t.regex(output, new RegExp(`Updated data.`));
+});
+
+// read_stale_data
+test.serial(`should read stale data from an example table`, (t) => {
+  t.plan(2);
+  // read-stale-data reads data that is exactly 10 seconds old.  So, make sure
+  // 10 seconds have elapsed since the update_data test.
+  return new Promise((resolve, reject) => {
+    setTimeout(async () => {
```

Comment:
Nit: can you promisify the `setTimeout()` call, and wrap the tests in a `then` as was done [here](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/6fca8cf07d951b9731acd4615310edb8b3e10ef3/pubsub/system-test/topics.test.js#L73-L79)?

Reply:
Done.